wordpress-mobile/WordPress-iOS
# Empty state image: use image natural sizes

Fixes #n/a

Previously, the empty state image was set to a hard width of 239. This made some images bigger/smaller than they were supposed to be. This removes the image constraints, allowing the images to be their natural size.

To test:
Go to several empty state views. Verify the images appear properly. 

Example images that are now sized correctly:
- no Blog Posts.
- no Site Pages.
- Notifications > no notifications.

![blog_posts](https://user-images.githubusercontent.com/1816888/48224564-60c54100-e357-11e8-8e25-938fc95037cf.png)

![site_pages](https://user-images.githubusercontent.com/1816888/48224635-a1bd5580-e357-11e8-95c7-cbc58f2510f0.png)

![notifications](https://user-images.githubusercontent.com/1816888/48224703-dd581f80-e357-11e8-9a06-bb0227cf6e7a.png)
